include/internal/quic_error.h

```diff
@@ -35,7 +35,44 @@
 
 /* Inclusive range for handshake-specific errors. */
 #  define QUIC_ERR_CRYPTO_ERR_BEGIN          0x0100
-#  define QUUC_ERR_CRYPTO_ERR_END            0x01FF
+#  define QUIC_ERR_CRYPTO_ERR_END            0x01FF
+
+#  define QUIC_ERR_CRYPTO_ERR(X) \
+    (QUIC_ERR_CRYPTO_ERR_BEGIN + (X))
+
+#  define QUIC_ERR_CRYPTO_MISSING_EXT \
+    QUIC_ERR_CRYPTO_ERR(TLS13_AD_MISSING_EXTENSION)
+
+#  define QUIC_ERR_CRYPTO_NO_APP_PROTO \
+    QUIC_ERR_CRYPTO_ERR(TLS1_AD_NO_APPLICATION_PROTOCOL)
+
+static ossl_inline ossl_unused const char *
+ossl_quic_err_to_string(uint64_t error_code)
```

Comment:
Not sure if a header is the best place for this.
It keeps it with the errors which is a plus, but it seems odd to have error handling as an inline function.

Similarly for quic_wire.h's frame_type_to_string.